src/applications/caregivers/components/FormFields/FacilityList.jsx

```diff
@@ -93,7 +93,7 @@ const FacilityList = props => {
 };
 
 FacilityList.propTypes = {
-  value: PropTypes.string.isRequired,
+  value: PropTypes.string,
   facilities: PropTypes.array,
```

Comment:
Lets fix this linting warning as well.

Reply:
Just this last warning to fix. `onChange` needs to be at the bottom of the list.